.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "deployment/maykin-deployment"]
-	path = deployment/maykin-deployment
-	url = git@bitbucket.org:maykinmedia/maykin-deployment.git
```

Comment:
I'm not sure if we want to remove `maykin-deployment`?

Reply:
This was done as proposed by @alextreme :-)